Source/WebCore/page/Quirks.h

```diff
@@ -25,9 +25,10 @@
 
 #pragma once
 
+#include "ElementTargetingTypes.h"
```

Comment:
Can we forward declare?

Reply:
Hm.. it's a bit tricky since `TargetedElementSelectors` is a typedef, but I can simply use `Vector<HashSet<String>>` instead to avoid the `#include` in `Quirks.h`.

Thanks for the review!